packages/serverless/src/apollo/app/rollup.ts

```diff
@@ -22,7 +22,7 @@ enum RollUpState {
 type RollUpHistoryItem {
   databaseName: String!
   transactionType: TransactionType!
-  transactionHash: String,
```

Comment:
Make sure we only use `txHash` in our codebase

Reply:
done